internal/publish/publish_test.go

```diff
@@ -736,6 +736,8 @@ func TestPublishWithBypass(t *testing.T) {
 						}
 
 						if tc.HealthAuthority != nil {
+							// statistics are written in the background.
+							time.Sleep(2 * time.Second)
```

Comment:
Can we make this 5s just to avoid flakes? I'd rather have a slightly longer test than have to re-run it 4x.